qlik.dev/tutorials/platform-operations/sdk-python/index.jinja

```diff
@@ -2,65 +2,65 @@
 <html>
 
 <head>
+    <link rel="icon" href="data:,">
     <script src="https://unpkg.com/enigma.js/enigma.min.js"></script>
 </head>
 
 <body>
 
 <div id="main">
+    <p>Content embedded from tenant: <b>{{ TENANT_HOSTNAME }} </b></p>
     <iframe id='qlik_frame' style='border:none;width:100%;height:900px;'></iframe>
 </div>
 
 <script>
 
         //    CONFIGURATION
 
-        const TENANT = '{{ TENANT_HOSTNAME }}';
-        const JWTENDPOINT = '{{ JWT_URL }}';
-        const WEBINTEGRATIONID = '{{ WEB_INTEGRATION_ID }}';
-        const APPID = '{{ APP_ID }}';
-        const SHEETID = '{{ SHEET_ID }}';
+        const TENANT_HOSTNAME = '{{ TENANT_HOSTNAME }}';
+        const JWT_ENDPOINT = '{{ JWT_URL }}';
+        const WEB_INTEGRATION_ID = '{{ WEB_INTEGRATION_ID }}';
+        const APP_ID = '{{ APP_ID }}';
+        const SHEET_ID = '{{ SHEET_ID }}';
+        const APP_OPEN_RETRY_COUNT = 120;
 
         //    MAIN
 
         (async function main() {
             const isLoggedIn = await qlikLogin();
             const qcsHeaders = await getQCSHeaders();
-            const [session, enigmaApp] = await connectEnigma(qcsHeaders, APPID);
+            const [session, enigmaApp] = await connectEnigma(qcsHeaders, APP_ID);
             handleDisconnect(session);
             const theme = await getTheme(enigmaApp);
-            renderSingleIframe('qlik_frame', APPID, SHEETID, theme);
+            renderSingleIframe('qlik_frame', APP_ID, SHEET_ID, theme);
         })();
 
         //    LOGIN
 
         async function qlikLogin() {
-            const loggedIn = await checkLoggedIn();
-            if (loggedIn.status !== 200) {
-                const tokenRes = await (await getJWTToken(JWTENDPOINT)).json();
-                const loginRes = await jwtLogin(tokenRes.body);
-                if (loginRes.status != 200) {
-                    const message = 'Something went wrong while logging in.';
-                    alert(message);
-                    throw new Error(message);
-                }
-                const recheckLoggedIn = await checkLoggedIn();
-                if (recheckLoggedIn.status !== 200) {
-                    const message = 'Third-party cookies are not enabled in your browser settings and/or browser mode.';
-                    alert(message);
-                    throw new Error(message);
-                }
+            const tokenRes = await (await getJWTToken(JWT_ENDPOINT)).json();
```

Comment:
This forces a new JWT and corresponding login on every page load - makes it much easier to see what's going on in an full flow.